docs/docs/self-service.mdx

```diff
@@ -118,6 +118,13 @@ Ory Kratos as well.
 
 <SelfServiceBrowserFlow flows={['login', 'settings', '...']} />
 
+:::info
```

Comment:
I think this is burried quite deep in the documentation. Is there maybe a better place to track this? For example in Debug & Help? I think most people struggling with this will probably not find this.